assignment.js

```diff
@@ -32,6 +32,13 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let multiply = num => {
```

Comment:
@SOyeniyi801 nice use of using a function here (even though we hadn't yet gone over functions at this point)